### PR TITLE
Add Personal Uses

### PR DESCRIPTION
**Pull request by kemitchell**
This PR adds a new section, analogous to Noncommercial Organizations, to create a "safe harbor" for listed personal uses. The idea is to help sharpen the lines of "noncommercial purpose" and increase licensee confidence.

For those who may have seen earlier, nonpublic drafts, this language has been condensed from its original list form, like Noncommercial Organizations.

### CODE REVIEW

**Comment by kemitchell** [comment]
@heathermeeker we shared concerns, in earlier drafts, about the lengths of both the organizational and personal "safe harbors" for noncommercial purpose. When I shortened up Noncommercial Organizations, I also took a hand at shortening Personal Uses.

I'm on the fence about whether we should include something like this.

**Comment by heathermeeker** [comment]
I will think it is unnecessary to enumerate, but we might say any use for an individual (as opposed to a company) is non-commercial. All the enumerated  uses are probably fair use anyway.

**Comment by kemitchell** [comment]
@heathermeeker I think allowing any personal use would go too far. Both because some personal uses are the definition of commercial, and also because we'd merely beg the line drawing problem of differentiating personal action for personal benefit and personal action on behalf of, or for the benefit of, another organization.

File: polyform.md [hunk 1]
```diff
@@ -26,14 +26,16 @@ The licensor grants you a patent license for the software that covers patent cla
 
 Any noncommercial purpose is a permitted purpose.
 
+## Personal Uses
+
+Personal use for research, experiment, and testing for the benefit of public knowledge, without any anticipated commercial application, personal study, private entertainment, hobby projects, amateur pursuits, or religious observance is for a permitted purpose.
```
Comment (karianna) [inline polyform.md]: The wording seems awkward here. I'd have the `...anticipated commercial application...` wording last.

Comment (kemitchell) [reply polyform.md]: Addressed in 1474eeb4c666942458f5dbc5682c1b6048e4787a

**Pull request merged**